src/main/java/entities/User.java

```diff
@@ -81,6 +79,9 @@ private List<Object> extractReviewBody(Review review){
         return revBody;
     }
 
+    public void like(String accName){
+        this.liked.add(accName);
+    }
```

Comment:
Maybe add a method to extract a list of all Users class using the list of usernames